staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1/types.go

```diff
@@ -52,12 +52,20 @@ type ExecCredentialSpec struct {
 }
 
 // ExecCredentialStatus holds credentials for the transport to use.
+//
+// Most of the fields below are sensitive. This data should only be transmitted
```

Comment:
This comment only applies to the `Token` & `ClientKeyData` fields, right?

Reply:
Yep, clarified